Source/SwiftyDropbox/Platform/SwiftyDropbox_iOS/OAuthMobile.swift

```diff
@@ -60,10 +88,18 @@ open class DropboxMobileOAuthManager: DropboxOAuthManager {
         }
         
         if let scheme = dAuthScheme(sharedApplication) {
-            let nonce = UUID().uuidString
-            UserDefaults.standard.set(nonce, forKey: kDBLinkNonce)
-            UserDefaults.standard.synchronize()
-            sharedApplication.presentExternalApp(dAuthURL(scheme, nonce: nonce))
+            let url: URL
+            if let authSession = authSession {
+                // Code flow
+                url = dAuthURL(scheme, authSession: authSession)
+            } else {
+                // Token flow
+                let nonce = UUID().uuidString
+                UserDefaults.standard.set(nonce, forKey: kDBLinkNonce)
+                UserDefaults.standard.synchronize()
```

Comment:
Synchronize is not required. https://developer.apple.com/documentation/foundation/nsuserdefaults/1414005-synchronize?language=objc

`https://developer.apple.com/documentation/foundation/nsuserdefaults/1414005-synchronize?language=objc`